app/views/photos/_photo_comments.html.erb

```diff
@@ -0,0 +1,39 @@
+<div id="comments-container">
```

Comment:
If your comments are polymorphic, use the same comment partial as you did for posts to keep code DRY.